gatsby-config.js

```diff
@@ -2,5 +2,20 @@ module.exports = {
   siteMetadata: {
     title: 'Gatsby Default Starter',
   },
-  plugins: ['gatsby-plugin-react-helmet'],
+  plugins: [
+    'gatsby-plugin-react-helmet',
+    {
+      resolve: `gatsby-plugin-manifest`,
+      options: {
+        name: 'gatsby-starter-default',
+        short_name: 'starter',
+        start_url: '/',
+        background_color: '#f7f0eb',
+        theme_color: '#a2466c',
+        display: 'minimal-ui',
+        // icon: 'src/images/icon.png', // This path is relative to the root of the site.
```

Comment:
You want to just add the Gatsby logo? https://github.com/gatsbyjs/gatsby/blob/master/www/src/assets/gatsby-icon.png